xds/src/test/java/io/grpc/xds/ClusterImplLoadBalancerProviderTest.java

```diff
@@ -44,6 +45,8 @@ public void provided() {
 
   @Test
   public void providesLoadBalancer()  {
+    TlsContextManagerImpl unused =
```

Comment:
This is useless and should be deleted. You should not take shortcut for the code. If you found something is untestable or awkward for testing, that means the class design/dependency is problematic. We should correct the design instead of bring in shortcut to the code and keep going wrong direction.

Reply:
I have deleted the `unused` declaration (agree it was useless). But I need this call to initialize the `TlsContextManagerImpl` singleton instance otherwise the test will cause the code to try to load the actual bootstrap file.